apps/Testkit.java/Testkit.java.android/Tests/AndroidClient2/app/src/main/java/com/couchbase/androidclient/MainActivity.java

```diff
@@ -0,0 +1,128 @@
+package com.couchbase.androidclient;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import com.couchbase.lite.*;
+import java.util.Date;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.TimerTask;
+import java.util.Random;
+
+public class MainActivity extends AppCompatActivity {
+
+    private Database database;
+    private Document doc;
+    private Replicator replicator;
+    private int numOfDocs = 4;
+    private int scenarioRunTimeMinutes = 1;
+    private String syncGatewayURL = "blip://192.168.33.11:4984/db/";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        /*syncGatewayURL = System.getProperty("syncGatewayURL");
+        numOfDocs = Integer.parseInt(System.getProperty("numOfDocs"));
+        scenarioRunTimeMinutes = Integer.parseInt(System.getProperty("scenarioRunTimeMinutes"));*/
+
+        numOfDocs = getIntent().getIntExtra("numOfDocs",4);
+        scenarioRunTimeMinutes = getIntent().getIntExtra("scenarioRunTimeMinutes",1);
+        syncGatewayURL = getIntent().getStringExtra("syncGatewayURL");
+
+        setContentView(R.layout.activity_main);
+        DatabaseConfiguration config = new DatabaseConfiguration(this);
+
+        Log.i("state","Creating database");
+        Log.i("app","printing  check --- : ");
+        Log.i("app"," printing  1 --- : + %s 2 : %s 3 : %s", numOfDocs ,  scenarioRunTimeMinutes , syncGatewayURL);
```

Comment:
oops sorry, I forgot to remove second statement, I just added for some debugging. Third statement prints the values of parameter that I passed from shell script to android app .

Let me once again check to remove unnecessary code.